README.md

```diff
@@ -305,6 +305,70 @@ Run the tool through the context menu on a file or directory:
 
 </details>
 
+### Workflow for GitHub Actions
+
+<details>
+
+#### What does it do?
+
+This workflow will convert every JPG, JPEG and PNG file into AVIF and WebP.
+
+#### Add Workflow
+
+Add the following file in the following location:
+`.github/workflows/optimizt.yml`
+
+##### Push changes trough commit
+
+The current workflow will push changes based on commit. This means when anything happens with a JPG/JPEG/PNG it will be triggerd.
+
+Insert the following into optimizt.yml if you want this process fully automatic.
+```yml
+name: optimizt
+on:
+  # Triggers the workflow on push or pull request events but only for the main branch and only when there's JPG/JPEG/PNG in the commmit!
+  push:
+    branches: [main]
+    paths:
+      - "**.jpe?g"
+      - "**.png"
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+jobs:
+  run-optimizt:
+    runs-on: ubuntu-latest
+    env:
+      OPTIMIZTCONVERTERARGS: --verbose --force --avif --webp . # convert to avif and webp for all JPG/JPEG/PNG files in this folder
+    steps:
+      # This fix "Missing write access to /usr/local/lib/node_modules" error
+      - name: properly configure node
+        uses: actions/setup-node@v2
+        with:
+          node-version: 14
+      - name: Install dependencies
+        run: | # install optimizt
+          npm i -g @funboxteam/optimizt
+      - uses: actions/checkout@v2 # This is a premade github action
+        with:
+          persist-credentials: false # otherwise, the token used is the GITHUB_TOKEN, instead of your personal token
+          fetch-depth: 0 # otherwise, you will failed to push refs to dest repo
+      - name: run optimizt
```

Comment:
If would be great if we use the same case for steps titles. “Run Optimizt” instead of “run optimizt” here, and “Properly configure Node.js” instead of “properly configure node” above 😺